yarn start throws an error

### Is this a bug report?

yep


### Environment


1. `node -v`:  v6.2.2
2. `npm -v`: yarn
3. `yarn --version` (if you use Yarn): 1.3.2
4. `npm ls react-scripts` (if you haven’t ejected): 1.3.0

Then, specify:

1. Operating system: OS X
2. Browser and version (if relevant): -


### Steps to Reproduce


```sh
create-react-app project-name --scripts-version netguru-react-scripts
cd project-name
yarn start
```

results in:
```
$ react-scripts start
module.js:442
    throw err;
    ^

Error: Cannot find module 'react-hot-loader/webpack'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.resolve (internal/module.js:27:19)
    at Object.<anonymous> (/Users/justynarachowicz/Projects/project-name/node_modules/netguru-react-scripts/config/webpack.config.dev.js:170:23)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/justynarachowicz/Projects/project-name/node_modules/netguru-react-scripts/scripts/start.js:41:16)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

--- a/packages/react-scripts/package.json
+++ b/packages/react-scripts/package.json
@@ -74,7 +74,7 @@
   "devDependencies": {
     "react": "^16.0.0",
     "react-dom": "^16.0.0",
-    "react-hot-loader": "next",
+    "react-hot-loader": "^3.1.3",
     "normalize.css": "^7.0.0"
   },
   "optionalDependencies": {

--- a/packages/react-scripts/scripts/init.js
+++ b/packages/react-scripts/scripts/init.js
@@ -101,7 +101,7 @@ module.exports = function(
     command = 'npm';
     args = ['install', '--save', verbose && '--verbose'].filter(e => e);
   }
-  args.push('react', 'react-dom', 'react-hot-loader@next', 'normalize.css');
+  args.push('react', 'react-dom', 'react-hot-loader@^3.1.3', 'normalize.css');
 
   // Install additional template dependencies, if present
   const templateDependenciesPath = path.join(

